src/core/raster/qgsrasterdataprovider.h

```diff
@@ -510,8 +510,7 @@ class CORE_EXPORT QgsRasterDataProvider : public QgsDataProvider, public QgsRast
     //! \note not available in Python bindings
     static QStringList cStringList2Q_( char **stringList ) SIP_SKIP;
 
-    static QString makeTableCell( const QString &value );
-    static QString makeTableCells( const QStringList &values );
+    static QString makeHtmlBulletList( const QStringList &values );
```

Comment:
Can this methods be private? Possibly even a generic `QgsHtmlUtils` class would be nice to create the same kind of lists throughout QGIS.
Mainly it shouldn't be exposed by the raster data api.

Reply:
Yes, better! It's fixed.

Reply:
A lot better, thanks!